pkg/java/jar/sonatype/sonatype.go

```diff
@@ -153,7 +153,7 @@ func (s Sonatype) SearchBySHA1(sha1 string) (jar.Properties, error) {
 	}, nil
 }
 
-func (s Sonatype) SearchByArtifactID(artifactID string) (string, error) {
+func (s Sonatype) SearchByArtifactID(artifactID, _ string) (string, error) {
```

Comment:
In Trivy we use `trivy-java-db`.
Tell me if you want to update the sonatype logic as well.